src/utils/options.f90

```diff
@@ -51,6 +51,15 @@ module options
         ! where k_max is the maximum x or y wavenumber and zeta_char is
         ! a characteristic vorticity (see subroutine adapt of strat.f90).
         ! Note: nnu = 3 and prediss = 10 are recommended.
+
+        ! Prefactor type to use:
+        ! - vorch: characteristic vorticity
+        ! - roll-mean-gmax: rolling mean of gamma_max
+        character(len=16) :: pretype = 'vorch'
+
+        ! Window size for the rolling mean approach
+        integer :: win_size = 1000
```

Comment:
Again, this could be a job parameter: roll-mean-win-size

Reply:
Yes, this is a job parameter. I can rename to "roll_mean_win_size".

Reply:
Done.